qiskit/compiler/assembler.py

```diff
@@ -46,7 +46,7 @@ def assemble(
         Schedule,
         List[Schedule],
         ScheduleBlock,
-        Union[ScheduleBlock],
+        ScheduleBlock,
```

Comment:
```suggestion
        List[ScheduleBlock],
```
pretty sure the `Union` was the wrong word, rather than being unnecessary.